src/plugins/overlay/overlay.js

```diff
@@ -60,7 +60,7 @@ var componentName = "wb-overlay",
 				footer = $elm.find( ".modal-footer" )[ 0 ];
 
 				var hasFooter = ( footer && footer.length !== 0 ) ? true : false,
-					hasButton = hasFooter && $( footer ).find( closeClass ).length !== 0,
+					hasButton = hasFooter && $( footer ).find( “.” + closeClass ).length !== 0,
```

Comment:
Can you use the vertical double quote?
```suggestion
					hasButton = hasFooter && $( footer ).find( "." + closeClass ).length !== 0,
```